lib/course.rb

```diff
@@ -1 +1,27 @@
+require './lib/student'
 
+class Course
+  attr_reader :name, :capacity, :students
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @students = []
+    @full = false
+  end
+
+  def full?
+    if @students.size < @capacity
+      @full = false
+    else
+      @full = true
+    end
+  end
+
+  def enroll(student)
+    if @students.size < @capacity
```

Comment:
```suggestion
    if !full?
```
We can use our `full?` method here instead of rewriting the logic.